src/rev_ai/__init__.py

```diff
@@ -3,4 +3,5 @@
 
 __version__ = '2.6.1'
 
-from .models import Job, JobStatus, Account, Transcript, MediaConfig, CaptionType
+from .baseclient import BaseClient
```

Comment:
BaseClient should be internal only. Does this syntax make it so that customers can access BaseClient?

Reply:
I only made this change in order to make imports be from `.` rather than from `.baseclient` in `apiclient.py` and `customvocabulariesclient.py`. But I see your point in keeping it internal.

In the same vein, would we want to put the other three api clients in this __init__ for easier access?